src/core_plugins/kibana/public/home/components/tutorial/saved_objects_installer.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  EuiSteps,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiButton,
+  EuiSpacer,
+  EuiCallOut,
+} from '@elastic/eui';
+
+export class SavedObjectsInstaller extends React.Component {
+  state = {
+    isInstalling: false,
+    isInstalled: false,
+  };
+
+  installSavedObjects = async () => {
+    this.setState({
+      isInstalling: true,
+    });
+
+    const resp = await this.props.bulkCreate(this.props.savedObjects, { overwrite: true });
+    const errors = resp.savedObjects.filter(savedObjectCreateResult => {
+      return savedObjectCreateResult.hasOwnProperty('error');
+    });
+    const hasErrors = errors.length > 0;
+
+    const statusMsg = hasErrors
+      ? `Unable to load kibana saved objects, Error: ${errors[0]}`
+      : `${this.props.savedObjects.length} saved objects successfully added`;
+
+    this.setState({
+      isInstalling: false,
+      installStatusMsg: statusMsg,
+      isInstalled: !hasErrors,
+    });
+  }
+
+  renderInstallMessage() {
+    if (!this.state.installStatusMsg) {
+      return;
+    }
+
+    return (
+      <EuiCallOut
+        title={this.state.installStatusMsg}
+        color={this.state.isInstalled ? 'success' : 'warning'}
+      />
+    );
+  }
+
+  renderInstallStep = () => {
+    const installStep = (
+      <Fragment>
+        <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+          <EuiFlexItem>
+            <EuiText>
+              <p>
+                Imports index pattern, visualizations and pre-defined dashboards.
+                Index pattern is required for some features in the APM UI.
```

Comment:
I'm thinking we don't want to call out APM specifically here since any module could use this.  Maybe we either nix this sentence or let tutorials customize?